sim/src/tlv.rs

```diff
@@ -294,6 +299,15 @@ impl TlvGen {
             ..Default::default()
         }
     }
+
+    #[allow(dead_code)]
+    pub fn new_hw_rollback_protection() -> TlvGen {
```

Comment:
I think `new_security_counter` or `new_sec_cnt` is more appropriate.
If you agree, please update all occurrences.